apnsworker/src/main/scala/apnsworker/models/ApnsException.scala

```diff
@@ -0,0 +1,58 @@
+package apnsworker.models
+
+import java.time.LocalDateTime
+import java.util.UUID
+
+
+sealed trait ApnsException extends RuntimeException {
```

Comment:
I didn't actually find a place where you'd throw the `ApnsException`. Assuming it's not thrown, should that be renamed `ApnsError` and should it stop inheriting from `RuntimeException`?

Reply:
Yes you are right 👍 . I change it and keep it inheriting from RuntimException